pages/players/[playerName].js

```diff
@@ -1,20 +1,68 @@
 import { useRouter } from 'next/router';
 import Navbar from '../../components/Navbar'
 import Footer from '../../components/Footer'
+import React from "react";
 
 
 
 const PlayerDetails = () => {
     
     const router = useRouter();
     const {playerName} = router.query;
```

Comment:
change this to replace the hyphens with a space and set it to uppercase rather than the lines 12-19 below

Reply:
what import does that?